fares-data-build-tool/create-data
# feat: CFD-444 export page design

# Description

-   Explain the purpose of this pr

# Testing instructions

- auth against test
- run 
1. `awslocal s3api list-objects --bucket fdbt-netex-data-dev | jq ".Contents[].Key"`
2.`awslocal s3 cp s3://fdbt-netex-data-dev/BLAC/carnetFlatFare.xml s3://fdbt-netex-data-dev/BLAC/exports/TestExport/carnet.xml`
3. `awslocal s3 cp s3://fdbt-netex-data-dev/BLAC/carnetFlatFare.xml s3://fdbt-matching-data-dev/BLAC/exports/TestExport/carnet.xml`
- go to exports page and check it looks how it should on figma

# Type of change
